src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java

```diff
@@ -1211,8 +1209,8 @@ protected boolean isInvalidSample(double sampleRate) {
         return sampleRate != 1 && ThreadLocalRandom.current().nextDouble() > sampleRate;
     }
 
-    boolean isOriginDetectionEnabled(String containerID, boolean originDetectionEnabled) {
-        if (!originDetectionEnabled || (containerID != null && !containerID.isEmpty())) {
+    boolean isOriginDetectionEnabled(boolean originDetectionEnabled) {
```

Comment:
Nit: now that the `containerID` isn't a parameter, the signature looks strange:
```
function is origin enabled(bool origin enabled)
```

Reply:
I agree, but I don't have better names. Hopefully it isn't too confusing in the broader context.